docs/src/index.md

```diff
@@ -41,22 +41,32 @@ Further worked examples of using this package can be found in the [DynamicHMCExa
 
 # General interface
 
+## Transformations
+
 ```@docs
 dimension
 transform
 transform_and_logjac
 ```
 
+## Inverses
+
 ```@docs
 inverse
 inverse!
 inverse_eltype
```

Comment:
Unrelated to this PR, but should `inverse_and_logjac` be added here?

Reply:
Good catch, opened #140 and will fix later.